ui/src/app/pages/signed-in/nav-bar.tsx

```diff
@@ -202,20 +156,7 @@ export const NavBar = () => {
     <Breadcrumb/>
     {window.location.pathname !== '/access-renewal' && <AccessRenewalNotificationMaybe/>}
     {window.location.pathname !== '/data-access-requirements' && <LoginGovIAL2NotificationMaybe/>}
-    {
-      showStatusAlert && <StatusAlertBanner
-          title={statusAlertDetails.title}
-          message={statusAlertDetails.message}
-          footer={
-            statusAlertDetails.link &&
-            <Button data-test-id='status-banner-read-more-button'
-                    onClick={() => window.open(statusAlertDetails.link, '_blank')}>
-              READ MORE
-            </Button>
-          }
-          onClose={onStatusAlertBannerUnmount}
-      />
-    }
+    <StatusAlertBanner/>
```

Comment:
note: does not always render anything.  should this be `StatusAlertBannerMaybe` instead?